### DISCUSSION TITLE
FeedViewState is always null with C# markup

### DISCUSSION THREAD

**Discussion by LuisAlfredo92** [unoplatform/uno#15089] [Q&A] [answered]
I tried to follow [some examples](https://platform.uno/docs/articles/external/workshops/tube-player/modules/09-FeedView-None-and-Error-templates/README.html?tabs=figma#add-feedview) with different data, but FeedViewState is always null, so I can't do anything
![image_2024-01-16_18-20-52](https://github.com/unoplatform/uno/assets/17792760/d397d39f-d9ba-4d6f-bd95-2c928072c804)

**Comment by dr1rrb** [comment, marked as answer]
Hey @LuisAlfredo92, this is because the `feedViewState` is provided only to write "strongly typed" binding, it will always be `null` (cf. Warning tip [here](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Markup/Templates.html#datatemplates))

In your sample you have to write `.ItemsSource(() => feedViewState.Data)`

Closing the issue, feel free to re-open it if you still have an issue :)

**Reply by LuisAlfredo92** [reply]
Oooohh thank you so much, I read the link you shared and after reading the warning I started reading Binding 101 and comprehend how it works

It's done now! Thanks!
![imagen](https://github.com/unoplatform/uno/assets/17792760/0d8ac177-f3f4-464b-9a96-aea43c836b01)
